### ISSUE TITLE
Ember 2.0-beta Ember.ArrayPolyfills Error

### ISSUE THREAD

**Issue by al3x-edge** [emberjs/data#3543]
Assuming it's due to this commit: https://github.com/emberjs/ember.js/pull/11458.  Apologies if this is reported elsewhere. Wasn't able to find anything.  Tried with `#beta` ember-data, but it's still point to `beta.19.1`. 

Versions:
Ember 2.0-beta2
Ember Data 1.13.5

```
Uncaught TypeError: Cannot read property 'indexOf' of undefined(anonymous function)
@ fixture-adapter.js:10(anonymous function) 
@ main.js:175
```

**Comment by wecc** [comment]
Hey @al3x-edge!

Ember Data 1.13 does not support Ember 2.0 because of IE8 support being removed in Ember 2.0. There will be a Ember Data 2.0 beta release any day now that will support Ember 2.0.

**Comment by al3x-edge** [comment]
@wecc Works for me. :smile: -- Closing then.

**Comment by tpitale** [comment]
@wecc any update on the Ember Data 2.0 beta plan that you know of? I see a PR by @bmac for the changelog updates.

**Comment by wecc** [comment]
@tpitale not yet, sorry

**Comment by truenorth** [comment]
Maybe we should start testing on a PR branch? It would suck for ember 2.0 to come out of beta, having not as much as it could have been, because many meaningful use cases involve ember data

**Comment by taras** [comment]
I'm with @mike-north, having Ember.js 2.0 without Ember Data 2.0 will be very limiting for any Ember.js 2.0 adopters.

**Comment by taras** [comment]
In either case, this issue should probably be reopened and tagged with Ember 2.0

**Comment by rwjblue** [comment]
@mike-north  / @taras:

**tldr;** You can absolutely test Ember 2.0.0-beta's or Ember canary today, with prebuilt Ember Data versions.

You can test Ember Data with Ember 2.0.0-beta's or canary, by using the auto-published canary builds of Ember Data.  What is now canary will become ember-data@2.0.0-beta.1 (or whatever they end up going with).

**Comment by al3x-edge** [comment]
@rwjblue Apologies if you answer this often; Where might the auto-published canary builds of Ember Data be? I didn't happen to see them on emberjs.com or in the releases on Github. Didn't notice a canary branch either.

**Comment by rwjblue** [comment]
> Where might the auto-published canary builds of Ember Data be?

They are auto-published to https://github.com/components/ember-data/tree/canary, which you can get via `bower install --save ember-data#canary`.

> I didn't happen to see them on emberjs.com or in the releases on Github.

They are listed here: http://emberjs.com/builds/#/canary (use these if you are not using bower for example)

**Comment by al3x-edge** [comment]
Oh. Feeling silly. Thanks @rwjblue -- I'll switch that over.
